db/schema.rb

```diff
@@ -55,8 +55,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "admin", default: false
-    t.string "photo_url"
-    t.index ["email"], name: "index_users_on_email", unique: true
+    t.string "photo_url"    t.index ["email"], name: "index_users_on_email", unique: true
```

Comment:
Here is where it looks like a new line was taken out -- can you add back in please?